components/home/ChartBox/Chart.tsx

```diff
@@ -123,16 +124,55 @@ const Chart = (props: ChartProps) => {
         snapTooltipToDatumX
         snapTooltipToDatumY
         showDatumGlyph
-        applyPositionStyle={true}
+        showVerticalCrosshair
+        verticalCrosshairStyle={{ stroke: '#B8B8B8', strokeDasharray: 3 }}
+        showSeriesGlyphs
+        applyPositionStyle
         style={{
           background: 'transparent',
         }}
         unstyled={false}
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         renderTooltip={({ tooltipData }: any) => (
-          <div className="bg-tooltipBackground px-[8px] py-[4px] rounded-[8px] text-baseForeground text-12">
-            {label && <p className="text-10 text-neutral-200">{label}</p>}
-            {AmountConverter(tooltipData.nearestDatum.datum.count)}
+          <div className="bg-tooltipBackground px-[8px] py-[4px] rounded-[8px] text-neutral-200 text-xs">
+            <span className="text-[10px]">
+              {dayjs
+                .utc(tooltipData.nearestDatum.datum.date)
+                .local()
+                .format('dddd, D MMMM YYYY')
+                .toString()}
+            </span>
+            {currentPeriod && prevPeriod ? (
+              <div>
+                <div className="flex items-center mr-20">
+                  <span
+                    className={`w-[0.25rem] md:w-[0.375rem] h-[0.25rem] md:h-[0.375rem] mr-5 rounded-full bg-secondary-500`}></span>
+                  Current:
+                  <span className="text-baseForeground pl-1">
+                    {AmountConverter(
+                      tooltipData.datumByKey['Current Week'].datum.count,
```

Comment:
update this key too.
As explained in the prev period

Reply:
fixed